src/quo2/screens/switcher_tabs.cljs

```diff
@@ -0,0 +1,9 @@
+(ns quo2.screens.switcher-tabs
```

Comment:
i don't think there is any sense to have it in the quo2 sandbox its not like component, its more like fundamental part of application